doc/client_lua_api.txt

```diff
@@ -609,6 +609,8 @@ Helper functions
     * Converts a string representing an area box into two positions
 * `minetest.is_yes(arg)`
     * returns whether `arg` can be interpreted as yes
+* `minetest.is_nan(arg)`
+    * returns true if passed argument is an number and number is NaN.
```

Comment:
\* a number
Shorter suggestion: `Returns true when the passed number represents NaN.`

Reply:
You don't need to define the function in C++.
```Lua
function minetest.is_nan(arg)
	return type(arg) == "number" and arg ~= arg
end
```

Reply:
i need it because core engine itself should not crash like a coward because a mod owner has forgotten this line.